modules/aruco/include/opencv2/aruco.hpp

```diff
@@ -336,7 +336,7 @@ class CV_EXPORTS_W GridBoard : public Board {
  */
 CV_EXPORTS_W int estimatePoseBoard(InputArrayOfArrays corners, InputArray ids, Ptr<Board> &board,
```

Comment:
You should make sure that the extra defined parameters here also get assigned a `@param` explanation above!